### DISCUSSION TITLE
ttkbootstrap script converted to executable does not run.

### DISCUSSION THREAD

**Discussion by kanwar-rajput** [israel-dryer/ttkbootstrap#369] [Q&A]
After I convert my script into executable using `cx_freeze`. I get this errors when I run the executable file.

      Traceback (most recent call last):
        File "C:\Users\KANWAR\AppData\Local\Programs\Python\Python310\Lib\site-packages\cx_Freeze\initscripts\__startup__.py", line 120, in run
          name = name.replace(char, "_")
        File "C:\Users\KANWAR\AppData\Local\Programs\Python\Python310\Lib\site-packages\cx_Freeze\initscripts\Console.py", line 16, in run
          exec(code, module_main.__dict__)  # pylint: disable=exec-used
        File "main.py", line 207, in <module>
        File "C:\Users\KANWAR\AppData\Local\Programs\Python\Python310\lib\site-packages\ttkbootstrap\window.py", line 273, in __init__
          self._style = Style(themename)
        File "C:\Users\KANWAR\AppData\Local\Programs\Python\Python310\lib\site-packages\ttkbootstrap\style.py", line 488, in __init__
          localization.initialize_localities()
        File "C:\Users\KANWAR\AppData\Local\Programs\Python\Python310\lib\site-packages\ttkbootstrap\localization\msgs.py", line 9, in initialize_localities
          m.initialize()
        File "C:\Users\KANWAR\AppData\Local\Programs\Python\Python310\lib\site-packages\ttkbootstrap\localization\msgs.py", line 27, in initialize
          MessageCatalog.set_many(self.locale, *messages)
        File "C:\Users\KANWAR\AppData\Local\Programs\Python\Python310\lib\site-packages\ttkbootstrap\localization\msgcat.py", line 142, in set_many
          return int(root.tk.eval(out))
      _tkinter.TclError: invalid command name "::msgcat::mcmset"

Is there any solution for this man? It will be really appreciated~

**Comment by TheDemonGuard** [comment]
> cx_freeze



> After I convert my script into executable using `cx_freeze`. I get this errors when I run the executable file.
> 
> ```
>   Traceback (most recent call last):
>     File "C:\Users\KANWAR\AppData\Local\Programs\Python\Python310\Lib\site-packages\cx_Freeze\initscripts\__startup__.py", line 120, in run
>       name = name.replace(char, "_")
>     File "C:\Users\KANWAR\AppData\Local\Programs\Python\Python310\Lib\site-packages\cx_Freeze\initscripts\Console.py", line 16, in run
>       exec(code, module_main.__dict__)  # pylint: disable=exec-used
>     File "main.py", line 207, in <module>
>     File "C:\Users\KANWAR\AppData\Local\Programs\Python\Python310\lib\site-packages\ttkbootstrap\window.py", line 273, in __init__
>       self._style = Style(themename)
>     File "C:\Users\KANWAR\AppData\Local\Programs\Python\Python310\lib\site-packages\ttkbootstrap\style.py", line 488, in __init__
>       localization.initialize_localities()
>     File "C:\Users\KANWAR\AppData\Local\Programs\Python\Python310\lib\site-packages\ttkbootstrap\localization\msgs.py", line 9, in initialize_localities
>       m.initialize()
>     File "C:\Users\KANWAR\AppData\Local\Programs\Python\Python310\lib\site-packages\ttkbootstrap\localization\msgs.py", line 27, in initialize
>       MessageCatalog.set_many(self.locale, *messages)
>     File "C:\Users\KANWAR\AppData\Local\Programs\Python\Python310\lib\site-packages\ttkbootstrap\localization\msgcat.py", line 142, in set_many
>       return int(root.tk.eval(out))
>   _tkinter.TclError: invalid command name "::msgcat::mcmset"
> ```
> 
> Is there any solution for this man? It will be really appreciated~



> After I convert my script into executable using `cx_freeze`. I get this errors when I run the executable file.
> 
> ```
>   Traceback (most recent call last):
>     File "C:\Users\KANWAR\AppData\Local\Programs\Python\Python310\Lib\site-packages\cx_Freeze\initscripts\__startup__.py", line 120, in run
>       name = name.replace(char, "_")
>     File "C:\Users\KANWAR\AppData\Local\Programs\Python\Python310\Lib\site-packages\cx_Freeze\initscripts\Console.py", line 16, in run
>       exec(code, module_main.__dict__)  # pylint: disable=exec-used
>     File "main.py", line 207, in <module>
>     File "C:\Users\KANWAR\AppData\Local\Programs\Python\Python310\lib\site-packages\ttkbootstrap\window.py", line 273, in __init__
>       self._style = Style(themename)
>     File "C:\Users\KANWAR\AppData\Local\Programs\Python\Python310\lib\site-packages\ttkbootstrap\style.py", line 488, in __init__
>       localization.initialize_localities()
>     File "C:\Users\KANWAR\AppData\Local\Programs\Python\Python310\lib\site-packages\ttkbootstrap\localization\msgs.py", line 9, in initialize_localities
>       m.initialize()
>     File "C:\Users\KANWAR\AppData\Local\Programs\Python\Python310\lib\site-packages\ttkbootstrap\localization\msgs.py", line 27, in initialize
>       MessageCatalog.set_many(self.locale, *messages)
>     File "C:\Users\KANWAR\AppData\Local\Programs\Python\Python310\lib\site-packages\ttkbootstrap\localization\msgcat.py", line 142, in set_many
>       return int(root.tk.eval(out))
>   _tkinter.TclError: invalid command name "::msgcat::mcmset"
> ```
> 
> Is there any solution for this man? It will be really appreciated~

The error message seems similar to the one reported on the following discussion.
However, the executable was generated via pyinstaller tool.
https://github.com/israel-dryer/ttkbootstrap/discussions/318

Is it possible to try with the latest version?
If the issue still persists, please provide steps/instructions to reproduce the issue along with a demo/sample script.
